lib/tiktok/getProfile.js

```
@@ -0,0 +1,27 @@
+import getActorStatus from "../apify/getActorStatus.js";
+import getDataset from "../apify/getDataset.js";
+import { STEP_OF_ANALYSIS } from "../step.js";
+import getFormattedAccountInfo from "./getFormattedAccountInfo.js";
+
+const getProfile = async (datasetId, chat_id) => {
+  try {
+    while (1) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)